spark-launch/src/main/scala/com/ibm/sparktc/sparkbench/sparklaunch/confparse/ConfigWrangler.scala

```diff
@@ -89,49 +95,21 @@ object ConfigWrangler {
     }
   }
 
-
   /**
-    * The one config file needs to be split up into individual files that can be submitted to spark-submit. This
-    * @param rootConf
-    * @param oneSparkSubmitConf
+    * Utility method to simplify extracting the spark-submit configs from the original config object
+    * @param rootConfig
     * @return
     */
-  private[sparklaunch] def writePartialConfToDisk(rootConf: Config, oneSparkSubmitConf: Config): String = {
-    val tmpFile = Files.createTempFile("spark-bench-", ".conf")
-    // Write this particular spark-context to a separate file on disk.
-    val sbConf = stripSparkSubmitConfPaths(rootConf, oneSparkSubmitConf)
-    val output = sbConf.root.render(concise.setJson(false))
-    Files.write(tmpFile, output.getBytes(), StandardOpenOption.CREATE, StandardOpenOption.TRUNCATE_EXISTING)
-    tmpFile.toString
+  private[sparklaunch] def getListOfSparkSubmits(rootConfig: Config): Seq[Config] = {
+    getConfigListByName(s"${SLD.sparkSubmitObject}", rootConfig)
```

Comment:
Is there a reason to use string interpolation here?

Reply:
Nope, I think that's leftover from when it was a bigger combination of strings